products/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from products.views import ProductCategoryView, ProductListView
+
+urlpatterns = [
+    # http://127.0.0.1:8000/products/categories/1
+    path('/categories/<int:category_id>', ProductCategoryView.as_view()),
+
+    # http://127.0.0.1:8000/products/categories/list
+    path('/categories/list', ProductListView.as_view())
```

Comment:
`list`를 굳이 적어주지 않아도 되겠습니다.
```suggestion
    path('/categories', ProductListView.as_view())
```

Reply:
넵! 해당 부분 수정하겠습니다.
수정 : path('/categories', ProductListView.as_view())